data/json/itemgroups/locations.json

```diff
@@ -78,4 +78,3 @@
     ]
   }
 ]
- 
```

Comment:
So in my local version its showing that line 81 was deleted and line 82 is the new line 81. But here its showing that just line 81 was deleted and there is no line 82. Weird.